src/layout/header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import './Header.styled.scss'
+
+import { useLocation, useNavigate } from 'react-router-dom'
+
+import { RoutesManager } from '../../routesManager'
+import { Logo } from '../../ui-base/logo/Logo'
+
+export const Header = () => {
+  const navigate = useNavigate()
+  const { pathname } = useLocation()
+
+  return (
+    <header className="container">
+      <Logo />
+      <div className="buttonsWrapper">
+        <button
+          className={pathname === '/home' ? 'active' : ''}
+          onClick={() => navigate(RoutesManager.home.root.getURL())}
```

Comment:
I think it will be better to move it to a separate function **`handleClick`**
```suggestion
          onClick={handleClick}
```

Reply:
Good idea. Done